index.js

```diff
@@ -4,6 +4,8 @@ import { Provider } from 'react-redux'
 import store from './src/state/store/store'
 import { registerRootComponent } from 'expo'
 
+global.store = store
```

Comment:
what is this??

Reply:
That is equivalent to window.store in React!